src/substrate/filters/enricher.ts

```diff
@@ -455,6 +455,110 @@ export async function Enrich(
         };
       }
 
+      case EventKind.TreasuryBountyProposed: {
+        const [ bountyIndex ] = event.data as unknown as [ BountyIndex ] & Codec;
+        const bountyOpt = await api.query.treasury.bounties<Option<Bounty>>(bountyIndex);
+        if (!bountyOpt.isSome) {
+          throw new Error(`could not fetch treasury proposal index ${+bountyIndex}`);
```

Comment:
May want to change this copy to "treasury bounty index".

Reply:
I named it after the treasury interface type:
`/** @name BountyIndex */ export interface BountyIndex extends u32 {}`